src/pudl/package_data/ferc1/dbf_to_xbrl.csv

```diff
@@ -3215,62 +3215,62 @@ f1_respdnt_control,24,1994,Line 24,,,,
-f1_retained_erng,21,1994,Line 21,,,,
-f1_retained_erng,22,1994,TOTAL Appropriations of Retained Earnings (Acct. 436),,,,
-f1_retained_erng,23,1994,Dividends Declared-Preferred Stock (Account 437),,,,
-f1_retained_erng,24,1994,Line 24,,,,
-f1_retained_erng,25,1994,Line 25,,,,
-f1_retained_erng,26,1994,Line 26,,,,
-f1_retained_erng,27,1994,Line 27,,,,
-f1_retained_erng,28,1994,Line 28,,,,
-f1_retained_erng,29,1994,TOTAL Dividends Declared-Preferred Stock (Acct. 437),,,,
-f1_retained_erng,30,1994,Dividends Declared-Common Stock (Account 438),,,,
-f1_retained_erng,31,1994,Line 31,,,,
-f1_retained_erng,32,1994,Line 32,,,,
-f1_retained_erng,33,1994,Line 33,,,,
-f1_retained_erng,34,1994,Line 34,,,,
-f1_retained_erng,35,1994,Line 35,,,,
-f1_retained_erng,36,1994,TOTAL Dividends Declared-Common Stock (Acct. 438),,,,
-f1_retained_erng,37,1994,"Transfers from Acct 216.1, Unapprop. Undistrib. Subsidiary Earnings",,,,
-f1_retained_erng,38,1994,"Balance - End of Year (Total 1,9,15,16,22,29,36,37)",,,,
-f1_retained_erng,39,1994,Line 39,,,,
-f1_retained_erng,40,1994,Line 40,,,,
-f1_retained_erng,41,1994,Line 41,,,,
-f1_retained_erng,42,1994,Line 42,,,,
-f1_retained_erng,43,1994,Line 43,,,,
-f1_retained_erng,44,1994,Line 44,,,,
-f1_retained_erng,45,1994,TOTAL Appropriated Retained Earnings (Account 215),,,,
-f1_retained_erng,46,1994,"TOTAL Approp. Retained Earnings-Amort. Reserve, Federal (Acct. 215.1)",,,,
-f1_retained_erng,47,1994,"TOTAL Approp. Retained Earnings (Acct. 215, 215.1) (Total 45,46)",,,,
-f1_retained_erng,48,1994,"TOTAL Retained Earnings (Account 215, 215.1, 216) (Total 38, 47)",,,,
-f1_retained_erng,49,1994,Balance-Beginning of Year (Debit or Credit),,,,
-f1_retained_erng,50,1994,Equity in Earnings for Year (Credit) (Account 418.1),,,,
-f1_retained_erng,51,1994,(Less) Dividends Received (Debit),,,,
-f1_retained_erng,52,1994,Other Changes (Explain),,,,
-f1_retained_erng,53,1994,Balance-End of Year (Total lines 49 thru 52),,,,
-f1_retained_erng,1,2004,Balance-Beginning of Period,,,,
-f1_retained_erng,38,2004,"Balance - End of Period (Total 1,9,15,16,22,29,36,37)",,,,
-f1_retained_erng,48,2004,"TOTAL Retained Earnings (Acct. 215, 215.1, 216) (Total 38, 47) (216.1)",,,,
+f1_retained_erng,1,1994,Balance-Beginning of Year,reported_value,unappropriated_retained_earnings_previous_year,,
+f1_retained_erng,2,1994,Changes,header,,,
+f1_retained_erng,3,1994,Adjustments to Retained Earnings (Account 439),header,#N/A,,
+f1_retained_erng,4,1994,Credit:,reported_value,adjustments_to_retained_earnings_credit_unstructured,,
+f1_retained_erng,5,1994,Credit:,reported_value,adjustments_to_retained_earnings_credit_unstructured,,
+f1_retained_erng,6,1994,Credit:,reported_value,adjustments_to_retained_earnings_credit_unstructured,,
+f1_retained_erng,7,1994,Credit:,reported_value,adjustments_to_retained_earnings_credit_unstructured,,
+f1_retained_erng,8,1994,Credit:,reported_value,adjustments_to_retained_earnings_credit_unstructured,,
```

Comment:
I labeled all of these chunks of rows that correspond to the unstructured table with the suffix of `_unstructured`. The stem here corresponds to the stem in the "total-ish" row in dbf and the xbrl stem for the total factiod in the structured  table and the stem for all of the columns in the unstructured table as well

Reply:
Sorry my discomfort about non-unique factoids below was referring to rows like with identical `_unstructured` factoids, which I think would result in data loss (or an error?) if they aren't processed preemptively in a particular way, and which (I think?) would prevent us from distinguishing between the distinct rows once the row alignment has happened. Maybe we just need to talk about this a little. Is there a reason why they couldn't have unique names and `_unstructured` suffix?

```
adjustments_to_retained_earnings_credit_row4_unstructured
adjustments_to_retained_earnings_credit_row5_unstructured
adjustments_to_retained_earnings_credit_row6_unstructured
adjustments_to_retained_earnings_credit_row7_unstructured
adjustments_to_retained_earnings_credit_row8_unstructured
```

Reply:
Okay we're not really dealing with the unstructured parts right now -- we'll have another conversation about how these labels work when we get there.